tests/models/unets/test_models_unet_spatiotemporal.py

```diff
@@ -36,6 +37,7 @@
 enable_full_determinism()
 
 
+@skip_mps
```

Comment:
More than 60% of the tests under this class fails on my Mac Silicon (because Silicon currently doesn't have support for Conv3D). Better to just skip them entirely.